hadoop-hdds/rocksdb-checkpoint-differ/pom.xml

```diff
@@ -68,11 +68,21 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <groupId>org.apache.ozone</groupId>
       <artifactId>hdds-test-utils</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.ozone</groupId>
+      <artifactId>ozone-common</artifactId>
+      <version>${ozone.version}</version>
```

Comment:
Please don't add dependency in `hadoop-hdds` for anything in `hadoop-ozone`.  Move code, if necessary.

Reply:
Removed `ozone-common` dependency and moved configs back to `OzoneConfigKeys` for now.